ncm-metaconfig/src/main/metaconfig/logstash/pan/config_5.0.pan

```diff
@@ -0,0 +1,13 @@
+unique template metaconfig/logstash/config_5.0;
+
+variable METACONFIG_LOGSTASH_VERSION = '5.0';
```

Comment:
this line prevent setting the version using a final variable and there is no value in setting this here. either use it as is on th enext line, or even better, since this is `5.0`, harcode the 5.0 schema below.
same for the other versions